### PR TITLE
924 orphaned pages list

### LINKED ISSUE #924: ✨New Orphaned Rules page (Rules without categories)
Cc: @bradystroud @adamcogan @danielmackay @pierssinclairssw

Hi Team
  
### Pain

Currently there is no easy way of seeing rules without a category tied to them.

### Suggested Solution
Create an orphaned rules page where all rules without a category can be seen

### Tasks

- [ ] Create an orphaned rules page
- [ ] Bottom of rules home page - Add link to new page

### Screenshots

image

Thanks!

### PR DESCRIPTION
**Pull request by BrookJeynes**
closes #924

### CODE REVIEW

File: src/pages/orphaned.js [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+                            </div>
+                          </div>
+                        </section>
+
+                        <section
+                          className={`rule-content mb-4
+                            ${selectedOption === 'all' ? 'visible' : 'hidden'}`}
+                        >
+                          <div
+                            dangerouslySetInnerHTML={{ __html: rule.html }}
```
Comment (danielmackay) [inline src/pages/orphaned.js:243]: It's not ideal that we have to use `dangerouslySetInnerHTML` here, but this is already being used by `rule.js` to render the majority of rules.

File: src/pages/orphaned.js [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+                          />
+                        </section>
+
+                        <section
+                          className={`rule-content mb-4
+                          ${selectedOption === 'blurb' ? 'visible' : 'hidden'}`}
+                        >
+                          <div
+                            dangerouslySetInnerHTML={{ __html: rule.excerpt }}
```
Comment (danielmackay) [inline src/pages/orphaned.js:252]: Likewise here

**Review by danielmackay — DISMISSED** [review]

File: src/pages/orphaned.js [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+import Breadcrumb from '../components/breadcrumb/breadcrumb';
+import { Link } from 'gatsby';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { faGithub } from '@fortawesome/free-brands-svg-icons';
+import {
+  faArrowCircleRight,
+  faPencilAlt,
+} from '@fortawesome/free-solid-svg-icons';
```
Comment (bradystroud) [inline src/pages/orphaned.js:14]: Consolidate fontawesome imports
```suggestion
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { ApplicationInsights } from '@microsoft/applicationinsights-web';
import { faGithub } from '@fortawesome/free-brands-svg-icons';
import {
  faArrowCircleRight,
  faPencilAlt,
  faExclamationTriangle
} from '@fortawesome/free-solid-svg-icons';
```

**Review by bradystroud — APPROVED** [review]
🚀
One small change, looks great otherwise

File: src/pages/orphaned.js (hunk 1 above)
Comment (bradystroud) [reply src/pages/orphaned.js:243]: I have raised this concern before and we decided it was ok as the html being rendered comes from Markdown which has gone through the PR approval process.
This should ensure nothing malicious is rendered.

Hey @BrookJeynes 
1. Its probably a good idea to document this, most devs will have the same question.
2. Check the docs rule to find the best place for this.
3. Add a comment in the code with a link to docs wherever `dangerouslySetInnerHTML` is used

**Review by danielmackay — APPROVED** [review]

**Review by bradystroud — APPROVED** [review]

**Pull request merged**